processing/src/main/java/io/druid/query/groupby/having/HavingSpec.java

```diff
@@ -32,7 +32,8 @@
     @JsonSubTypes.Type(name = "not", value = NotHavingSpec.class),
     @JsonSubTypes.Type(name = "greaterThan", value = GreaterThanHavingSpec.class),
     @JsonSubTypes.Type(name = "lessThan", value = LessThanHavingSpec.class),
-    @JsonSubTypes.Type(name = "equalTo", value = EqualToHavingSpec.class)
+    @JsonSubTypes.Type(name = "equalTo", value = EqualToHavingSpec.class),
+    @JsonSubTypes.Type(name = "dimSelector", value = DimensionSelectorHavingSpec.class)
```

Comment:
missing documentation, and I don't fully see the advantage of adding this feature instead of using a filter, which will be much faster.